app/models/notification.rb

```diff
@@ -64,6 +64,8 @@ def self.concatenate_or_create(recipient, target, actor, notification_type)
       begin
         n.actors = n.actors | [actor]
         n.unread = true
+        # I have to specially touch it to update updated_at whenever new actor is inserted in notification.
```

Comment:
Don't use first person in comments. Either "We need to explicitly ..." or just  "Explicitly touch the notification ..."